lib/mayaUsd/commands/baseExportCommand.h

```diff
@@ -18,92 +18,90 @@
-    //
-    // The list of short forms of flags defined as Arg Tokens:
-    static constexpr auto kDefaultMeshSchemeFlag = "dms";
-    static constexpr auto kDefaultUSDFormatFlag = "duf";
-    static constexpr auto kExportColorSetsFlag = "cls";
-    static constexpr auto kExportUVsFlag = "uvs";
-    static constexpr auto kEulerFilterFlag = "ef";
-    static constexpr auto kExportVisibilityFlag = "vis";
-    static constexpr auto kExportInstancesFlag = "ein";
-    static constexpr auto kMergeTransformAndShapeFlag = "mt";
-    static constexpr auto kStripNamespacesFlag = "sn";
-    static constexpr auto kExportRefsAsInstanceableFlag = "eri";
-    static constexpr auto kExportDisplayColorFlag = "dsp";
-    static constexpr auto kShadingModeFlag = "shd";
-    static constexpr auto kMaterialsScopeNameFlag = "msn";
-    static constexpr auto kExportMaterialCollectionsFlag = "mcs";
-    static constexpr auto kMaterialCollectionsPathFlag = "mcp";
-    static constexpr auto kExportCollectionBasedBindingsFlag = "cbb";
-    static constexpr auto kNormalizeNurbsFlag = "nnu";
-    static constexpr auto kExportReferenceObjectsFlag = "ero";
-    static constexpr auto kExportSkelsFlag = "skl";
-    static constexpr auto kExportSkinFlag = "skn";
-    static constexpr auto kParentScopeFlag = "psc";
-    static constexpr auto kRenderableOnlyFlag = "ro";
-    static constexpr auto kDefaultCamerasFlag = "dc";
-    static constexpr auto kRenderLayerModeFlag = "rlm";
-    static constexpr auto kKindFlag = "k";
-    static constexpr auto kCompatibilityFlag = "com";
-    static constexpr auto kChaserFlag = "chr";
-    static constexpr auto kChaserArgsFlag = "cha";
-    static constexpr auto kMelPerFrameCallbackFlag = "mfc";
-    static constexpr auto kMelPostCallbackFlag = "mpc";
-    static constexpr auto kPythonPerFrameCallbackFlag = "pfc";
-    static constexpr auto kPythonPostCallbackFlag = "ppc";
-    static constexpr auto kVerboseFlag = "v";
-
-    // Short and Long forms of flags defined by this command itself:
-    static constexpr auto kAppendFlag = "a";
-    static constexpr auto kAppendFlagLong = "append";
-    static constexpr auto kFilterTypesFlag = "ft";
-    static constexpr auto kFilterTypesFlagLong = "filterTypes";
-    static constexpr auto kFileFlag = "f";
-    static constexpr auto kFileFlagLong = "file";
-    static constexpr auto kSelectionFlag = "sl";
-    static constexpr auto kSelectionFlagLong = "selection";
-    static constexpr auto kFrameSampleFlag = "fs";
-    static constexpr auto kFrameSampleFlagLong = "frameSample";
-    static constexpr auto kFrameStrideFlag = "fst";
-    static constexpr auto kFrameStrideFlagLong = "frameStride";
-    static constexpr auto kFrameRangeFlag = "fr";
-    static constexpr auto kFrameRangeFlagLong = "frameRange";
 
-    MStatus doIt(const MArgList& args) override;
-    bool  isUndoable () const override { return false; };
+    class MAYAUSD_CORE_PUBLIC MayaUSDExportCommand : public MPxCommand {
```

Comment:
Just noticed that we have class opening-braces on the same line - is this intentional? Don't care much personally, but it seemed a bit odd to have functions on a new line, but classes on the same, so wanted to confirm this was the intent...

Reply:
I believe this was inherited from `WebKit` and we never changed the default. Looking over the repo, we are pretty consistent in opening braces to be on a new line so I don't think anyone will disagree here.